cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -58,26 +58,12 @@ spec:
             - /heapster
             - --source=kubernetes.summary_api:''
             - --sink=gcm
-          volumeMounts:
-            - name: ssl-certs
```

Comment:
Does heapster not need to talk to googleapis.com?

Reply:
With this change heapster uses the SSL certs that are built into the container, rather than those on the host. My understanding is that the certificates in the container are sufficient that.

Reply:
(To clarify - usually this practice of mounting the host certificates is only needed for `FROM scratch` containers. However, heapster [copies in the host certs](https://github.com/kubernetes/heapster/blob/master/deploy/docker/Dockerfile) when it's built, so this isn't necessary).

Reply:
how long are those valid for?

Reply:
```
$ openssl x509 -in heapster/92474ba266e5a063e88a4cc5f136d5bac305cb75538b4086a73c1ac27d834ce6/etc/ssl/certs/ca-certificates.crt  -text
Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number: 6828503384748696800 (0x5ec3b7a6437fa4e0)
    Signature Algorithm: sha1WithRSAEncryption
        Issuer: CN=ACCVRAIZ1, OU=PKIACCV, O=ACCV, C=ES
        Validity
            Not Before: May  5 09:37:37 2011 GMT
            Not After : Dec 31 09:37:37 2030 GMT
        Subject: CN=ACCVRAIZ1, OU=PKIACCV, O=ACCV, C=ES
...
```
Looks like that shouldn't be a problem, as long as they aren't revoked.

That said, this is a a common requirement for containers. Maybe we need to build a solution that doesn't require a HostPath volume...

Reply:
Ok, ya. I agree.